sky/tools/create_full_ios_framework.py

```diff
@@ -168,9 +170,15 @@ def zip_archive(dst):
       'zip', '-r', 'artifacts.zip', 'gen_snapshot_arm64', 'Flutter.xcframework'
   ],
                         cwd=dst)
+  if (os.path.exists(os.path.join(dst, 'Flutter.dSYM', 'Contents'))):
+    subprocess.check_call([
+        'zip', '-r',
+        '%s/Flutter.dSYM.zip' % dst,
+        '%s/Flutter.dSYM/Contents' % dst
```

Comment:
Is this replacing something that will be moved from the recipe?  I see https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/engine/engine.py#1375 but it's zipping the `Flutter.dSYM`, not the `Contents` subdirectory.

Reply:
Nice catch, updated!

Reply:
Let me know when this is all checked in and I can validate the final output, if you want.

Reply:
Also maybe https://github.com/flutter/engine/blob/main/testing/symbols/verify_exported.dart should be updated to check this zip exists?

Reply:
I'm missing adding the verify exported check. Whether this script is run or not will depend on the configuration file. I may need to make some changes to verify_exported before I can add it to the config file. Let run it with led and share the build.

Reply:
Here is the led run https://chromium-swarm.appspot.com/task?id=5cad4aa97b75841

Reply:
Looks like an outage, I can take a look on Monday
![Screen Shot 2022-08-12 at 6 24 14 PM](https://user-images.githubusercontent.com/682784/184463319-91f921d5-02da-4fcf-84a5-60885b51bb13.png)

Reply:
Here is one successful led run: https://chromium-swarm.appspot.com/task?id=5cb1a2362e463c10

Reply:
How can I download the artifacts to check if the symbols are in the right place?

Reply:
You can click the `gsutil.upload` links at the end of the build, they will take you to the upload location in gcs.